src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    private static final Scanner SCANNER = new Scanner(System.in);
```

Comment:
this should better not be stored in a field

Reply:
done